tests/slepos/build_images_creator.pm

```diff
@@ -0,0 +1,256 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Build SLEPOS images using image creator
+# Maintainer: Pavel Sladek <psladek@suse.cz>
+
+use base "basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use slepos_images;
+
+
+my %template_done;
+
+sub build_image {
+
+    my ($image, $template, $template_idx, $packages_ar) = @_;
+
+    my ($imagebase, $version) = split_image_name($image);
+    bmwqemu::diag("image '$image' will be built using image creator:");
+
+    #add virtual drivers unless already done
+    if (!$template_done{$template}) {
+        select_console('root-console');
+        wait_still_screen;
+        save_screenshot;
+
+        prepare_template($template);
+        $template_done{$template} = 1;
+
+        select_console('x11');
+        wait_still_screen;
```

Comment:
that does not make sense to me. `select_console` already asserts the "x11" console so wait_still_screen should not be needed